package.json

```diff
@@ -79,10 +80,16 @@
     "onCommand:cmake.outline.buildAll",
     "workspaceContains:**/CMakeLists.txt",
     "workspaceContains:.vscode/cmake-kits.json"
-    ],
+  ],
   "main": "./dist/main",
   "contributes": {
     "commands": [
+      {
+        "command": "cmake.activeFolder",
+        "title": "Get active folder",
+        "when": "cmake:enableFullFeatureSet",
```

Comment:
You need to add 'when' to the menus/commandPalette list to make it effective (actually it does nothing in contributes/commands).

That being said, I'm not sure if enableFullFeatureSet is required here.